btf_interp/src/lib.rs

```diff
@@ -2,50 +2,199 @@
 use btf_types::BrainFuckProgram;
 use std::num::NonZeroUsize;
 
+pub trait CellKind {
+    fn wrapped_add(&mut self, v: &Self) -> Self;
+    fn wrapped_sub(&mut self, v: &Self) -> Self;
```

Comment:
Does Brainfuck need `wrapped_add()` and `wrapped_sub()` or could these be simpler? (eg. `wrapped_inc()` and `wrapped_dec()`)